AdzerkSDK/AdzerkSDK/AdzerkSDK.swift

```diff
@@ -499,6 +499,7 @@ public typealias ADZUserDBUserResponseCallback = (ADZUser?, Error?) -> ()
         let config = URLSessionConfiguration.ephemeral
         config.httpAdditionalHeaders = [
             "User-Agent" : UserAgentProvider.instance.userAgent,
+            "X-Adzerk-Sdk-Version": "adzerk-ios-sdk:\(AdzerkSDKVersion)",
```

Comment:
How does the SDK Version variable get set here?

Reply:
That is a static variable defined in the same file. We don't have any automation around this at the moment, it must be manually set.

Reply:
@subdigital Is there an "easy" way to automate it at build-time or anything? Also, I just realized that the key should be `adzerk-decision-sdk-ios` to match the pattern of the other SDKs (we're releasing both decision and management in the future so knowing which in the string is valuable).

Reply:
I can add a script to modify this file w/ the bundle version at build time. I'll also fix that key.